### PR TITLE
[DOCS] README rewrite

### PR DESCRIPTION
**Pull request by shikharish**
Updated README in new format

### CODE REVIEW

**Review by proffapt — CHANGES_REQUESTED** [review]
(back to top) sabse hatana nahi tha. Please follow the template.

**Review by proffapt — CHANGES_REQUESTED** [review]
Also add that `additional documentation`, take those docs from https://github.com/metakgp/README.md itself. As said already, stick to the template.

Also why is this metakgp wiki wala badge going to next line..
image

**Comment by harshkhandeparkar** [comment]
> Also why is this metakgp wiki wala badge going to next line..

It is due to the comment. Adding the license badge back will fix this.

File: README.md [hunk 1]
```diff
@@ -34,79 +77,134 @@ curl 'https://erp.iitkgp.ernet.in/Acad/timetable_track.jsp?action=second&dept=AE
...
+[contributors-shield]: https://img.shields.io/github/contributors/metakgp/chillzone.svg?style=for-the-badge
+[contributors-url]: https://github.com/metakgp/chillzone/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/metakgp/chillzone.svg?style=for-the-badge
+[forks-url]: https://github.com/metakgp/chillzone/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/chillzone.svg?style=for-the-badge
+[stars-url]: https://github.com/metakgp/chillzone/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/chillzone.svg?style=for-the-badge
+[issues-url]: https://github.com/metakgp/chillzone/issues
+<!-- [license-shield]: https://img.shields.io/github/license/shikhar-irez/chillzone?label=license&style=for-the-badge
+[license-url]: https://github.com/shikhar-irez/chillzone/blob/new-readme/LICENSE -->
```
Comment (harshkhandeparkar) [inline README.md:208]:
```suggestion
[license-url]: https://github.com/shikhar-irez/chillzone/blob/master/LICENSE -->
```

This branch will get deleted once merged.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,72 @@
...
-- [How does it work?](#how-does-it-work)
+<div id="top"></div>
+
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+<!-- [![MIT License][license-shield]][license-url] -->
```
Comment (harshkhandeparkar) [inline README.md:9]:
```suggestion
[![GPL v3][license-shield]][license-url]
```

**Comment by shikharish** [comment]
> Adding the license badge back will fix this

There is some issue in license badge. Someone please create the LICENSE file in the master branch so I can directly link it in the README

**Comment by shikharish** [comment]
> Also add that additional documentation

Right now, additional docs would only contain LICENSE right? 
Chillzone does not have any other docs mentioned in the template

**Comment by proffapt** [comment]
arey, security, issue template.. look into: https://github.com/metakgp/README.md/tree/main/.github

**Comment by harshkhandeparkar** [comment]
> > Adding the license badge back will fix this
> 
> There is some issue in license badge. Someone please create the LICENSE file in the master branch so I can directly link it in the README

Just add the link. It will work once the pr is merged.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,72 @@
...
+  </a>
+
+  <h1 align="center">Chillzone</h1>
+
+  <p align="center">
+    <i>Will find you a chillzone inside IIT KGP at any time</i>
+    <br />
+    <a href="https://github.com/metakgp/chillzone/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/metakgp/chillzone/issues">Request Feature</a>
```
Comment (harshkhandeparkar) [inline README.md:28]: Can replace one of these with website link and other one can be "Report Bug / Request Feature" or "Give Feedback"

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (shikharish) [reply README.md:28]: What should the text be for website?

Comment (harshkhandeparkar) [reply README.md:28]: Just "Website" or "Chillzone" or something. Do you have any suggestions?

Comment (shikharish) [reply README.md:28]: Webiste is okay:
image

Or pura link hi daal sakte hai:
image

Comment (harshkhandeparkar) [reply README.md:28]: website hi achha hai

**Review by proffapt — APPROVED** [review]
LGTM.

**Review by harshkhandeparkar — APPROVED** [review]

**Pull request merged**